lightning/src/chain/onchaintx.rs

```diff
@@ -1135,6 +1149,13 @@ impl<ChannelSigner: EcdsaChannelSigner> OnchainTxHandler<ChannelSigner> {
 				//- resurect outpoint back in its claimable set and regenerate tx
 				match entry.event {
 					OnchainEvent::ContentiousOutpoint { package } => {
+						// We pass 0 to `package_locktime` to get the actual required locktime.
+						let package_locktime = package.package_locktime(0);
```

Comment:
Uh, another bug found by the reorg test. 
Because `package_locktime` is lower-bounded by the current height, if we pass the current height to it, 
the package will be added to `locktimed_packages` no matter what actual locktime it requires.
That's why we need test coverage. :(

Reply:
Great! Feel free to close this one when ready.